actor.go

```diff
@@ -14,13 +14,14 @@ type ActorMaker interface {
 }
 
 func NewActorDef(name string) *ActorDef {
-	return &ActorDef{Name: name, Type: name, Settings: make(map[string]string)}
+	return &ActorDef{Name: name, Type: name, Settings: make(map[string]string), RawData: make(map[string][]byte)}
 }
 
 type ActorDef struct {
 	Name     string            `json:"name"`
 	Type     string            `json:"type"`
 	Settings map[string]string `json:"settings"`
+	RawData  map[string][]byte `json:"rawdata"`
```

Comment:
Let me check on the consequences of just making `Settings` be of type `map[string][]byte`. I think it will work, and it would keep the definition clearer.